packages/code-connect/components/Alert/OverflowFooter.figma.tsx

```diff
@@ -0,0 +1,42 @@
+import figma from '@figma/code-connect';
+import { AlertGroup } from '@patternfly/react-core';
+
+figma.connect(
+  AlertGroup,
+  'https://www.figma.com/design/aEBBvq0J3EPXxHvv6WgDx9/PatternFly-6--Components-Test?node-id=1110-2784',
+  {
+    props: {
+      // strings
+      overflowMessage: figma.string('✏️ Title'),
+
+      alertGroupOverflowContent: `{alerts.slice(0, maxDisplayed).map(({ key, variant, title }) => (
```

Comment:
what is this alerts object?

It needs to be defined in the code or it will get a `ReferenceError: alerts is not defined`

Reply:
@nicolethoen Opted for static alerts and added required props